### PR TITLE
CVAT Canvas updates

### PR DESCRIPTION
**Pull request by bsekachev**
* Based on MVC model
* Ability to setup image to canvas
* Zoom (wheel mouse)
* Fit (double click) 
* Move (mousedown + mousemove + mouseup)
* Base64 images instead of blobs in cvat-core (there are freezes if use blobs)

### CODE REVIEW

File: cvat-canvas/README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ All methods are sync.
  ```canvas_shape_drawing```
 - Tags has a class ```canvas_tag```
 - Canvas image has ID ```canvas_image```
+- Grid on the canvas has ID ```canvas_grid_pattern```
```
Comment (nmanovic) [inline cvat-canvas/README.md]: Does it make sense to have a prefix for all such elements like "cvat:canvas_grid_pattern"?

File: cvat-canvas/dist/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+window.addEventListener('DOMContentLoaded', async () => {
+    await window.cvat.server.login('admin', 'nimda760');
```
Comment (nmanovic) [inline cvat-canvas/dist/index.js]: remove 'admin' from here.

File: cvat-canvas/src/canvas.ts [hunk 3]
```diff
@@ -3,81 +3,88 @@
 * SPDX-License-Identifier: MIT
 */
 
-/* eslint-disable */
-// Temporary disable eslint
+import { CanvasModel, CanvasModelImpl, Rotation } from './canvasModel';
+import { CanvasController, CanvasControllerImpl } from './canvasController';
```
Comment (nmanovic) [inline cvat-canvas/src/canvas.ts]: Is it possible to have separate files for an interface and its implementation? Just want to keep interface files really small.

Comment (bsekachev) [reply cvat-canvas/src/canvas.ts]: @nmanovic 
There aren't any problem to do that, but It is not common practice for typescript as far as i know. 
Files with all interfaces (also called typings) will be automatically generated by a typescript compiler. They have extensions d.ts.

Comment (nmanovic) [reply cvat-canvas/src/canvas.ts]: OK

File: cvat-canvas/README.md (hunk 1 above)
Comment (bsekachev) [reply cvat-canvas/README.md]: @nmanovic 
What is a goal for that? To reduce a chance of id collision?

Comment (nmanovic) [reply cvat-canvas/README.md]: Yes.  It is the only purpose. It is a reason why C++ has namespaces. For example, canvas_image looks very general.

Comment (bsekachev) [reply cvat-canvas/README.md]: @nmanovic 
Okay, let's do it.

**Pull request merged**